packages/documentation/src/stories/internet-header/getting-started.stories.mdx

```diff
@@ -40,46 +42,63 @@ defineCustomElements();
 </body>
 ```
 
+#### Lazy-loaded
+
+```javascript
+// main.ts / index.js / ...
+import { defineCustomElements } from '@swisspost/internet-header/loader';
+
+defineCustomElements();
+```
+
+#### Bare component
+
+```javascript
+// main.ts / index.js / ...
+import '@swisspost/internet-header/dist/swisspost-internet-header/swisspost-internet-header.esm';
+```
+
 ### Include from a CDN
 
-If you are not using any loader or don't want to install from npm, you can load the `internet-header` from your favourite [CDN](https://en.wikipedia.org/wiki/Content_delivery_network).<br/>
-There are two options available. Make sure to replace `{version}` with the version you want to use.
+If you are not using any bundler or don't want to install from npm, you can load the `internet-header` from your favourite [CDN](https://en.wikipedia.org/wiki/Content_delivery_network).<br/>
+Make sure to replace `{version}` with the version you want to use or remove `@{version}` to use the latest version.
+
+Available CDNs:
 
-- [jsDelivr](https://www.jsdelivr.com/)
-- [unpkg](https://unpkg.com/)
+* [jsDelivr](https://www.jsdelivr.com/): replace `{cdnURL}` with [https://cdn.jsdelivr.net/npm/@swisspost/internet-header](https://cdn.jsdelivr.net/npm/@swisspost/internet-header).
+* [unpkg](https://unpkg.com/): replace `{cdnURL}` with [https://unpkg.com/@swisspost/internet-header](https://unpkg.com/@swisspost/internet-header).
 
-#### jsDelivr
+#### Lazy-loaded
+
+```html
+<!DOCTYPE html>
+<html>
+  <body>
+    <swisspost-internet-header project="your-project-id"></swisspost-internet-header>
+  </body>
+  <script type="module">
+    import { defineCustomElements } from '{cdnUrl}@{version}/loader/index.es2017.js';
+
+    defineCustomElements();
+  </script>
+</html>
+```
+#### Bare component
```

Comment:
Probably a good place to use the future tabs pattern.

Reply:
I opened a ticket for it https://github.com/swisspost/design-system/issues/1818